src/app/core/services/milestone.service.ts

```diff
@@ -24,7 +25,10 @@ export class MilestoneService {
   public fetchMilestones(): Observable<any> {
     return this.githubService.fetchAllMilestones().pipe(
       map((response) => {
-        this.milestones = this.parseMilestoneData(response);
+        const parsedMilestones = this.parseMilestoneData(response);
+        this.milestones = [...parsedMilestones];
+        this.milestones.splice(-3, 1);
+        this.altMilestones = parsedMilestones.slice(0, -2);
         this.hasNoMilestones = response.length === 0;
```

Comment:
Not sure why there needs to be this logic. Since we only want the actual milestones and not the default milestones, why not just do not append the two default milestones in `parseMilestoneData` at all? Instead of appending them and then slicing them off here.

Reply:
Also, not really sure what the `altMilestones` is for, other than this assignment of value, it is not used elsewhere.